iDiff/differs/pipDiff.go

```diff
@@ -86,11 +81,30 @@ func (d PipDiffer) getPackages(path string) (map[string]utils.PackageInfo, error
 					glog.Errorf("Could not find Python package %s for corresponding metadata info", packageName)
 					continue
 				}
+				currPackage := utils.PackageInfo{Version: version, Size: size}
+				addToMap(packages, packageName, pyVersion, currPackage)
+			}
 
-				packages[packageName] = utils.PackageInfo{Version: version, Size: size}
+			// if not package, check if Python file
```

Comment:
It is possible that this got jumbled in the merge but currently this is necessary for the unit tests to pass (without it python scripts are not found).  If you maybe see something I don't about how the indexing may have been changed let me know and I can try to backtrack.